ip_map_branch.txt

```diff
@@ -28,17 +28,17 @@ seven_c	https://github.com/openemr/openemr.git	rel-501	0	1	0	0	demo_5_0_0_5.sql
 seven_e	https://github.com/openemr/openemr.git	rel-502	0	0	1	0	0	0	1	seven.openemr.io/e	hey	branch	0	1	0	0	Public OpenEMR 5.0.2 Development Demo on Ubuntu 18.04
 seven_f	https://github.com/openemr/openemr.git	rel-502	0	0	0	0	demo_5_0_0_5.sql	0	1	seven.openemr.io/f	hey	branch	5.0.0	1	0	0	Public OpenEMR 5.0.2 Development Demo on Ubuntu 18.04 With Demo Data
-eight	https://github.com/openemr/openemr.git	master	0	1	0	0	0	0	1	eight.openemr.io	hey	branch	0	2	0	0	Public OpenEMR Development Demo on Alpine 3.12
-eight_a	https://github.com/openemr/openemr.git	master	0	1	0	0	demo_5_0_0_5.sql	0	1	eight.openemr.io/a	hey	branch	5.0.0	2	0	0	Public OpenEMR Development Demo on Alpine 3.12 With Demo Data
+eight	https://github.com/openemr/openemr.git	master	0	1	0	0	0	0	1	eight.openemr.io	hey	branch	0	2	0	0	OPEN
+eight_a	https://github.com/openemr/openemr.git	master	0	1	0	0	demo_5_0_0_5.sql	0	1	eight.openemr.io/a	hey	branch	5.0.0	2	0	0	OPEN
 eight_b	https://github.com/openemr/openemr.git	rel-600	0	0	0	0	0	0	1	eight.openemr.io/b	hey	branch	0	2	0	0	Public OpenEMR 6.0.0 Development Demo on Alpine 3.12
 eight_c	https://github.com/openemr/openemr.git	rel-600	0	0	0	0	demo_5_0_0_5.sql	0	1	eight.openemr.io/c	hey	branch	5.0.0	2	0	0	Public OpenEMR 6.0.0 Development Demo on Alpine 3.12 With Demo Data
 nine	https://github.com/openemr/openemr.git	master	0	1	1	0	0	0	1	nine.openemr.io	hey	branch	0	2	0	0	Public OpenEMR Development Demo on Ubuntu 20.04 (with nodejs 14)
 nine_a	https://github.com/openemr/openemr.git	master	0	1	0	0	demo_5_0_0_5.sql	0	1	nine.openemr.io/a	hey	branch	5.0.0	2	0	0	Public OpenEMR Development Demo on Ubuntu 20.04 (with nodejs 14) With Demo Data
-ten	https://github.com/openemr/openemr.git	master	0	1	0	0	0	0	1	ten.openemr.io	hey	branch	0	2	0	0	Public OpenEMR Development Demo on Alpine 3.9
-ten_a	https://github.com/openemr/openemr.git	master	0	1	0	0	demo_5_0_0_5.sql	0	1	ten.openemr.io/a	hey	branch	5.0.0	2	0	0	Public OpenEMR Development Demo on Alpine 3.9 With Demo Data
+ten	https://github.com/openemr/openemr.git	master	0	1	0	0	0	0	1	ten.openemr.io	hey	branch	0	2	0	0	Public OpenEMR Development Demo on Alpine 3.13
+ten_a	https://github.com/openemr/openemr.git	master	0	1	0	0	demo_5_0_0_5.sql	0	1	ten.openemr.io/a	hey	branch	5.0.0	2	0	0	Public OpenEMR Development Demo on Alpine 3.13 With Demo Data
 ten_b	https://github.com/openemr/openemr.git	master	0	0	0	0	0	0	1	ten.openemr.io/b	hey	branch	0	0	0	0	OPEN
-ten_c	https://github.com/openemr/openemr.git	rel-502	0	0	0	0	0	0	1	ten.openemr.io/c	hey	branch	0	1	0	0	Public OpenEMR 5.0.2 Development Demo on Alpine 3.9
-ten_d	https://github.com/openemr/openemr.git	rel-502	0	0	0	0	demo_5_0_0_5.sql	0	1	ten.openemr.io/d	hey	branch	5.0.0	1	0	0	Public OpenEMR 5.0.2 Development Demo on Alpine 3.9 With Demo Data
+ten_c	https://github.com/openemr/openemr.git	rel-600	0	0	0	0	0	0	1	ten.openemr.io/c	hey	branch	0	1	0	0	Public OpenEMR 6.0.0 Development Demo on Alpine 3.13
```

Comment:
In this case, would add this new demo Alpine 3.13, PHP 7.4 at https://www.open-emr.org/wiki/index.php/Development_6.0.0_Demo
(note the 3.12 is still the "primary" demo since 3.12 is what the docker of 6.0.0 is on)

And agree with dropping alpine 3.9 5.0 dev demo here:
https://www.open-emr.org/wiki/index.php/Development_5.0.2_Demo#Demo_.28Alpine_3.9_-_uses_php_7.2.29